add bytesizes excalidraw and content addressable data storage (#2718)
nf-core/website: sites/main-site/src/content/events/2024/bytesize_excalidraw.md

---
title: 'Bytesize: Excalidraw'
subtitle: Phil Ewels, Seqera
type: talk
startDate: '2024-09-17'
startTime: '13:00+02:00'
endDate: '2024-09-17'
endTime: '13:30+02:00'
locations:
  - name: Online
    links:
      - https://kth-se.zoom.us/j/68390542812
---

> In this nf-core bytesize session, we’ll dive into Excalidraw — a nifty tool that makes creating diagrams for your documentation a breeze. It's free, super easy to use, and, best of all, your files stay fully editable by anyone who picks them up. We’ll also touch on the VSCode plugin that integrates it right into your workflow, plus how you can work with both PNG and SVG formats. Stick around until the end for a few nerdy tricks, like how to make your diagrams play nice with dark mode.

Reviewer: ```suggestion
In this nf-core bytesize session, Phil will talk all about [Excalidraw](https://excalidraw.com/) — a nifty tool that makes creating diagrams for your documentation a breeze.

It's free, super easy to use, and, best of all, your files stay fully editable by anyone who picks them up. He'll also touch on the VSCode plugin that integrates it right into your workflow, plus how you can work with both PNG and SVG formats. Stick around until the end for a few nerdy tricks, like how to make your diagrams play nice with dark mode.
```
